samples/matter/weather_station/src/app_task.cpp

```diff
@@ -204,13 +176,15 @@ void AppTask::PostEvent(AppEvent::Type type, AppEvent::Handler handler)
 	PostEvent(&event);
 }
 
-int AppTask::SoftwareUpdateConfirmationHandler(uint32_t offset, uint32_t size, void *arg)
+#ifdef CONFIG_MCUMGR_SMP_BT
+void AppTask::RequestSMPAdvertisingStart(void)
 {
-	/* For now just print update progress and confirm data chunk without any additional checks. */
-	LOG_INF("Software update progress %d B / %d B", offset, size);
-
-	return 0;
+	AppEvent event;
+	event.mType = AppEvent::Type::kStartSMPAdvertising;
+	event.mHandler = [](AppEvent *) { GetDFUOverSMP().StartBLEAdvertising(); };
+	sAppTask.PostEvent(&event);
```

Comment:
We have a helper for that:

```suggestion
	sAppTask.PostEvent(AppEvent::Type::kStartSMPAdvertising, [](AppEvent *) { GetDFUOverSMP().StartBLEAdvertising(); });
```